velox/common/memory/tests/MemoryManagerTest.cpp

```diff
@@ -483,6 +483,17 @@ TEST(MemoryManagerTest, quotaEnforcement) {
   }
 }
 
+TEST(MemoryManagerTest, testCheckUsageLeak) {
+  FLAGS_velox_memory_leak_check_enabled = true;
+  auto& manager = MemoryManager::getInstance(
+      memory::MemoryManager::Options{.checkUsageLeak = false}, true);
+
+  auto rootPool = manager.addRootPool("duplicateRootPool", kMaxMemory);
+  auto leafPool = manager.addLeafPool("duplicateLeafPool", true);
+  ASSERT_EQ(rootPool->testingCheckUsageLeak(), false);
+  ASSERT_EQ(leafPool->testingCheckUsageLeak(), false);
```

Comment:
Could you also check the memory pool created through MemoryManager::addLeafPool? Thanks!

Reply:
I don't understand, this is exactly what this line checks